presto-docs/src/main/sphinx/admin/session-property-managers.rst

```diff
@@ -24,6 +25,65 @@ by default. All matching rules contribute to constructing a list of session prop
 are applied in the order they are specified. Rules specified later in the file override values
 for properties that have been previously encountered.
 
+
+For the database-based built-in manager, add an
+``etc/session-property-config.properties`` file with the following contents:
+
+.. code-block:: none
+
+    session-property-config.configuration-manager=db
+    session-property-manager.db.url=url
+    session-property-manager.db.username=username
+    session-property-manager.db.password=password
+    session-property-manager.db.refresh-period=50s
+
+Change the value of ``session-property-manager.db.url`` to the jdbc url of a mysql database.
+``session-property-manager.db.refresh-period`` should be set to how often presto refreshes to
+match the database.
+
+This database consists of three tables: session specs, client tags and properties.
```

Comment:
nit: let's mention the actual table names like `session_specs`, `session_client_tags`, and `session_property_values`